drivers/spi/spi_ll_stm32.c

```diff
@@ -723,7 +723,18 @@ static int transceive_dma(const struct device *dev,
 	/* This is turned off in spi_stm32_complete(). */
 	spi_stm32_cs_control(dev, true);
 
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+	/* SPI CFG1 register is write protected when SPI is enabled */
```

Comment:
We don't see direct link with the code and the comment, can you revisit ?

Reply:
comment changed to "* set request before enabling (else SPI CFG1 reg is write protected)"